app/controllers/admin_controller.rb

```diff
@@ -23,21 +21,7 @@ def show_rails_exceptions?
 
   # Expire cached attachment files for a request
   def expire_for_request(info_request)
-    # Clear out cached entries, by removing files from disk (the built in
-    # Rails fragment cache made doing this and other things too hard)
-    info_request.foi_fragment_cache_directories.each{ |dir| FileUtils.rm_rf(dir) }
-
-    # Remove any download zips
-    FileUtils.rm_rf(info_request.download_zip_dir)
-
-    # Remove the database caches of body / attachment text (the attachment text
-    # one is after privacy rules are applied)
-    info_request.clear_in_database_caches!
-
-    # also force a search reindexing (so changed text reflected in search)
-    info_request.reindex_request_events
-    # and remove from varnish
-    info_request.purge_in_cache
+    info_request.expire
```

Comment:
Only to preserve the symmetry of `expire_for_request` vs `expire_for_user`, which may well be a rubbish reason for keeping it